lib/scss-preprocessor/broccoli-import-theme-components.js

```diff
@@ -40,4 +49,23 @@ Import.prototype.addImports = function(content) {
   return content + importsString;
 };
 
+Import.prototype.addThemeComponentClasses = function(content) {
+  var inputPath = this.inputPaths[0];
+  var importFiles = walkSync(inputPath, { globs: this.include });
```

Comment:
@ianstarz I'm trying to figure out how to also get all of the parent theme's components here.

Currently this ends up something like:

```scss
// ui-tomato-theme/ui-tomato-theme.scss

@import "./components/tomato--ui-button--default";
@import "./components/tomato--ui-field--default";

@mixin ui-tomato-theme($theme...) {
  $theme: keywords($theme);

  &--ui-button--default {
    @include ui-button--default($theme);
  }

  &--ui-field--default {
    @include ui-field--default($theme);
  }
}
```

This is good but we also want to create the `$--component-name` classes for each of the components that are in the parent theme (the base theme). The goal is output a mixin that defines classes for the theme (ui-tomato-theme) for every component, whether it's defined in the parent theme or the sub theme.